boards/arm/mps2_an521/CMakeLists.txt

```diff
@@ -7,3 +7,85 @@
 zephyr_library()
 zephyr_library_sources(pinmux.c)
 zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
+
+if (CONFIG_BUILD_WITH_TFM)
+	set (TFM_ROOT_PATH $ENV{ZEPHYR_BASE}/../modules/tee/tfm/trusted-firmware-m)
+	set (MCUBOOT_DIR "${TFM_ROOT_PATH}/bl2/ext/mcuboot")
```

Comment:
`MCUBOOT_DIR`, should be set in some common place and be renamed to `TFM_MCUBOOT_DIR`